.github/workflows/commitMessage.yml

```diff
@@ -0,0 +1,31 @@
+name: Reusable Check Commit Message Workflow
+
+on:
+  workflow_call:
+    secrets:
+      CUSTOM_GITHUB_TOKEN:
+        required: true
```

Comment:
Consider using the `secrets: inherit` keyword in a caller workflow. For more infomation, see https://docs.github.com/en/actions/using-workflows/reusing-workflows#passing-secrets-to-nested-workflows

```suggestion
```